features/step_definitions/supplieraz_steps.rb

```diff
@@ -41,3 +41,54 @@
     ".//h2[@class='search-result-title']/a"
   ).text.should == normalize_whitespace(@supplier['name'])
 end
+
+Given(/I navigate to the list of '(.*)' page$/) do |value|
+  page.visit("#{dm_frontend_domain}/g-cloud/suppliers?prefix=#{value.split(' ').last}")
+  @data_store = @data_store || Hash.new
+  @data_store['supplier_alphabet'] = "#{value.split(' ').last}"
+end
+
+Then(/I am taken to page '(.*)' of results$/) do |page_number|
+  if current_url.include?('search?')
+    current_url.should include("#{dm_frontend_domain}/g-cloud/search?")
+    current_url.should include("page=#{page_number}")
+    current_url.should include("lot=iaas")
+    if page_number >= '2'
```

Comment:
The detail of this test _only_ works if you're on page 2, in fact (it checks that the 'next' page is page 3, I think?)

I think anyway it might be worth separating this into separate tests, such as "Then I can see a Next link to page 3" etc?